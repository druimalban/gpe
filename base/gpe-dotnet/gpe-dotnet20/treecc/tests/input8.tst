// literal code definition sections at end
a b
%%
%{ literal code %%} a b

%{embedded EOF
