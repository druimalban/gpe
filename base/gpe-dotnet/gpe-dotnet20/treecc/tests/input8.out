line 2: identifier (len = 1): a
line 2: identifier (len = 1): b
line 3: literal definitions at end:

%{ literal code %%} a b

%{embedded EOF

